content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -1285,3 +1285,37 @@ prisma studio --browser firefox
 ```terminal
 prisma studio --browser none
 ```
+
+### <inlinecode>package.json</inlinecode> entry options
+
+#### <inlinecode>schema</inlinecode> path
+
+The path to the desired `schema.prisma` file can be specified with the `prisma.schema` entry in the `package.json` file. The path defines the file the Prisma CLI should use when you run any of the CLI commands. Both absolute and relative paths are supported.
+
+```json file="package.json"
+{
+  "name": "my-project",
+  "version": "1.0.0",
+  "prisma": {
+    "schema": "./custom-path-to-schema/schema.prisma"
+  }
+}
+```
+
+This is available from version 2.7.0 and later.
+
+#### <inlinecode>seed</inlinecode> option
+
+The path to the file or script to populate a database with sample database is specified in the `prisma.seed` entry in the `package.json` file. The file specified is used when `prisma db seed` is invoked. Both absolute and relative paths are supported.
+
+See [Seeding your database](/guides/migrate/seed-database) <span class="guide"></span>
+
+```json file="package.json"
+{
+  "name": "my-project",
+  "version": "1.0.0",
+  "prisma": {
+    "seed": "node prisma/seed.js"
+  }
+}
+```
```

Comment:
Since when is this one available?

Reply:
I think we built that with the feature back in the day. @Jolg42 will know better next week.

Reply:
Found it. We released it in 3.0.1